charts/neon-proxy/templates/certificate.yaml

```diff
@@ -4,7 +4,7 @@ kind: Certificate
 metadata:
   name: {{ include "neon-proxy.certificate" . }}
 spec:
-  commonName: '{{ .Values.settings.domain }}'
+  commonName: '{{ .Values.settings.domain | quote }}'
```

Comment:
or `|quote` (best) or hardcoded quotes in templates, otherwise look what you will have

```console
% cat x-values.yaml                                
settings:
  domain: "just_a_string"

% helm template . -f x-values.yaml | grep -A6 Cert
kind: Certificate
metadata:
  name: just_a_string
spec:
  commonName: '"just_a_string"'
  dnsNames:
  - 'just_a_string'
```

Reply:
I'll fix it in separate PR